src/Action/ExceptionAction.php

```diff
@@ -80,6 +83,10 @@ public function __invoke(FlattenException $exception, Request $request): Respons
         $headers['X-Content-Type-Options'] = 'nosniff';
         $headers['X-Frame-Options'] = 'deny';
 
+        if ($exception->getStatusCode() == Response::HTTP_INTERNAL_SERVER_ERROR && !$this->debug) {
```

Comment:
@dunglas @dkarlovi , good point. I made a new commit to catch all server errors according the method isServerError of the Response Object.